src/nvim/api/ui.c

```diff
@@ -268,12 +290,12 @@ static void remote_ui_grid_resize(UI *ui, Integer grid,
                                   Integer width, Integer height)
 {
   Array args = ARRAY_DICT_INIT;
-  if (ui->ui_ext[kUINewgrid]) {
+  if (ui->ui_ext[kUIMultigrid]) {
```

Comment:
this change is not correct (already `ext_newgrid` uses `grid_resize`).

Reply:
Somehow terminal tests are failing after changing back to `kUINewgrid`. The problem is probably somewhere else. I'll see what I can find.